CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+This is a quick overview for what you should know when contributing to this Git repository.
+
+ - There is a code style guide in `arweave_styleguide.md`. Please note that we're using tabs for indentation.
+ - To run the tests you need to do two things:
+	1. Uncomment out `%-define(DEBUG, debug).` in `src/ar.hrl`.
+	2. Run `make test`.
+ - You can discuss development and get help from the Arweave organization and from the community in the `#dev` channel on [our Discord server](https://discord.gg/3UTNZky).
+ - The main Git repo is https://github.com/ArweaveTeam/arweave
+ - Branch out from `master` and send us a Pull Request back to our `master`. Make sure your branch is based on latest master before you create the PR. I.e, rebase on our master if needed.
```

Comment:
I would say "fork" since contributors would have to fork the repo. Also, it does not matter much which branch they choose.

Reply:
It matters which branch they branch out from and which they create the PR to.

How about this:
```
- Branch out from `master` and create a Pull Request back to `master` in the main repo. Make sure your branch is based on latest `master` before you create the PR. I.e, rebase on `master` in the main repo if needed.
```

Reply:
I checked the GitHub Glossary and it's a bit unclear what to call the main repo. Upstream repo? If so, then I'm suggesting this:

```
- Branch out from `master` and create a Pull Request back to `master` in the upstream repo. Make sure your branch is based on latest `master` before you create the PR. I.e, rebase on upstream if needed.
```

Reply:
The contributors do not have write access to the upstream repository, therefore they need to fork. And since the changes are in the fork, I think we are not concerned about the branch they use.

I would thus phrase it like:
```
Fork the repo, implement the changes in the fork, and open a pull request against the upstream master.
```

Reply:
I added a whole workflow section. How do you like it?

Reply:
Should be very clear now, thank you!